instance/models/mixins/openedx_config.py

```diff
@@ -311,3 +314,50 @@ def _get_configuration_variables(self):
             })
 
         return template
+
+    def _get_prometheus_variables(self):
+        """Get all Prometheus-related ansible variables specific to Open edX appservers."""
+        return {
+            **super()._get_prometheus_variables(),
+            'NODE_EXPORTER_NGINX_CONFIG_PATH': '/edx/app/nginx/sites-available/node-exporter',
+            'NODE_EXPORTER_HTPASSWD_PATH': '/edx/app/nginx/nginx.htpasswd',
+            'NODE_EXPORTER_USE_SSL': False,
+        }
+
+    def _get_consul_variables(self):
+        """Get all Consul-related ansible variables specific to Open edX appservers."""
+        return {
+            **super()._get_consul_variables(),
+            'consul_service_config': {
+                'service': [
+                    {
+                        "id": "edxapp-lms",
+                        "name": "edxapp",
+                        "port": 8000,
```

Comment:
I don't think it's useful to advertise the internal ports the services are running on.  The service catalogue is meant for service discovery by other servers, e.g. the load balancer, but no other server will send traffic to the internal port.  We should simply advertise edxapp on port 80 instead, without distinguishing LMS and CMS.  They are both part of the same HTTP service, and the difference between the two can only be expressed in terms of HTTP notions (e.g. the Host header), while the service catalogue is only about IP address and port.

Reply:
:+1: I was not entirely sure what to do either here so just chose these ports, thanks for clarifying.